schedule/qam/12-SP5/mau-filesystem@64bit.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: mau-filesystem
+description: Filesystem tests on Maintenance prepared image 
+vars:
+  COLLECT_COREDUMPS: '1'
+  EXTRATEST: prepare,filesystem
```

Comment:
Looks like you can remove these variables as you have schedules modules here properly.

Reply:
I didn't really check what is in yaml, but this is not needed, will remove it.

Reply:
obsolete EXTRATEST variable removed